modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -505,18 +505,30 @@ def test_astype_errors(errors):
     eval_general(modin_df, pandas_df, lambda df: df.astype("int", errors=errors))
 
 
-@pytest.mark.parametrize("has_dtypes", [False, True])
+@pytest.mark.parametrize(
+    "has_dtypes",
+    [
+        pytest.param(
+            False,
+            marks=pytest.mark.xfail(
```

Comment:
So we break HDK with the changes in this PR?

Reply:
It was a fix for the master branch, it has already been merged.